src/qt/locale/bitcoin_fr.ts

```diff
@@ -288,7 +288,7 @@
     </message>
     <message>
         <source>Show general overview of wallet</source>
-        <translation>Afficher une vue d’ensemble du porte-monnaie</translation>
+        <translation>Afficher une vue d&apos;ensemble du porte-monnaie</translation>
```

Comment:
Would "portfeuille" be better here? I think it's more consistent with the rest of the translations.

Reply:
It's pretty much the same thing: "porte-monnaie" and "portefeuille" are synonyms.

Reply:
"porte-monnaie" is more suitable in this case and is less redundant.